### ISSUE TITLE
Parse Error when using '>' inside blade expression in x-splade-lazy url attribute

### ISSUE THREAD

**Issue by majortom731** [protonemedia/laravel-splade#450]
- Laravel Version: 10.13.5
- PHP Version: 8.2.6
- Splade JS Version (npm): 1.4.14
- Splade PHP Version (composer): 1.4.14
- Dev environment (OS, Sail/Valet/etc): docker-compose

### Description:

I'm trying to use the <x-splade-lazy> component with the url attribute, like so: 
`<x-splade-lazy url="{{ route('lf_businesses/matching_customers', ['businessId'=>$business->id] ) }}">`

This fails with the exception:

syntax error, unexpected token "if" 

The generated code looks like this:
```
<?php $component->withAttributes(['url' => ''.e(route('lf_businesses/matching_customers', ['businessId'=>
@if(\ProtoneMedia\Splade\Facades\Splade::isLazyRequest())
$business->id] )).'']); ?>
```

Seems the `>` sign in the expression is breaking something...

### Steps To Reproduce Issue:

- create a view that uses the <x-splade-lazy> component
- in the url attribute, have blade eval an expression that contains a `>`

**Comment by majortom731** [comment]
Ok, I looked into it.

Seems like this HTML tag matching regexp in `CustomBladeCompiler` is not prepared for `>` inside HTML attributes:

```
    /**
     * Returns a regex pattern to match an HTML tag and its contents.
     */
    public static function regexForTag(string $tag): string
    {
        return '/(<\s*' . $tag . '[^>]*>)(.|\n)*?(<\/' . $tag . '>)/';
    }
```

**Comment by bardolf69** [comment]
So I see there has been a push to update this but it still apears to be a problem.  I'm trying to pass a variable to my route like so,
```<x-splade-lazy url="{{ route('company.news', [$company->symbol]) }}">``` which gives me the error ```syntax error, unexpected token "@", expecting identifier or variable or "{" or "$"```

**Comment by bardolf69** [comment]
ok I've tried every conceviable way of passing a route variable to the lazy-load url and none of them work.  Just for reference my setup is as follows.

- Laravel Version: 10.24.0
- PHP Version: 8.2.8
- Splade JS Version (npm): 1.4.16
- Splade PHP Version (composer): 1.4.16

**Comment by pascalbaljet** [comment]
Fixed in v1.4.19
